### PR TITLE
[SW-684] Disable URL tipping for URLs inside wallet

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

**Comment by CedrikNikita** [comment]
Check out how this feature works. Your code doesn't work for a redirect from `superhero.com`. 
https://feat-disabled-tips.wallet.z52da5wt.xyz/account?x-success=https%3A%2F%2Fsuperhero.com%2F&x-cancel=https%3A%2F%2Fsuperhero.com%2F

**Review by CedrikNikita — CHANGES_REQUESTED** [review]
Check out how this feature works. Your code doesn't work for a redirect from `superhero.com`. 
https://feat-disabled-tips.wallet.z52da5wt.xyz/account?x-success=https%3A%2F%2Fsuperhero.com%2F&x-cancel=https%3A%2F%2Fsuperhero.com%2F

File: src/popup/components/TransferSendForm.vue [hunk 1]
```diff
@@ -480,6 +486,16 @@ export default defineComponent({
       ) {
         root.$store.commit('accounts/setActiveIdx', mySignerAccounts[0].idx);
       }
+      const tipUrlEncoded: any = root.$route.query.url;
```
Comment (CedrikNikita) [inline src/popup/components/TransferSendForm.vue:489]: There are no more `url` in the query. And superhero is not providing such.

File: src/popup/components/TransferSendForm.vue [hunk 2]
```diff
@@ -419,6 +388,7 @@ export default defineComponent({
         if (!requestedTokenBalance) {
           formModel.value.address = '';
           openDefaultModal({ msg: root.$t('modals.insufficient-balance.msg') });
+          formModel.value.address = '';
```
Comment (subhod-i) [inline src/popup/components/TransferSendForm.vue:391]: `formModel.value.address` has been initialized twice.

**Review by subhod-i — APPROVED** [review]

**Review by pstachel — APPROVED** [review]

**Review by CedrikNikita — APPROVED** [review]
Code change look reasonable.

**Pull request merged**